osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyCalculator.cs

```diff
@@ -193,7 +221,7 @@ private double computeAimRating(double aimDifficultyValue, Mod[] mods, int total
             return aimRating * Math.Cbrt(ratingMultiplier);
         }
 
-        private double computeSpeedRating(double speedDifficultyValue, Mod[] mods, int totalHits, double approachRate, double overallDifficulty)
+        private double computeSpeedRating(double speedDifficultyValue, Mod[] mods, int totalHits, double approachRate, double overallDifficulty, double relevantMechanicalDifficulty)
```

Comment:
`relevantMechanicalDifficulty` is such a weird name. Relevant to what? Why not call it just `mechanicalDifficultyRating`?

Reply:
I don't really agree on "relevant" making it *any* clearer than it not being there. I think it's best we just remove it.

Reply:
I'd argue if we want to keep the `relevant` word in it then call it `visibilityRelevantMechanicalDifficulty` at least